src/snowcli/cli/snowpark/package/commands.py

```diff
@@ -97,23 +99,25 @@ def package_create(
         "-y",
         help="Install packages that are not available on the Snowflake anaconda channel",
     ),
-    **options,
-):
+    **kwargs,
```

Comment:
We have convention to name this as `**options`

Reply:
Done